src/zzz_od/gui/view/one_dragon/charge_plan_interface.py

```
@@ -364,10 +365,18 @@ def on_interface_shown(self) -> None:
         self.loop_opt.init_with_adapter(get_prop_adapter(self.config, 'loop'))
         self.skip_plan_opt.init_with_adapter(get_prop_adapter(self.config, 'skip_plan'))
         self.restore_charge_opt.init_with_adapter(get_prop_adapter(self.config, 'restore_charge'))
+        self._update_restore_charge_opt_status(self.config.skip_plan)
 
     def on_interface_hidden(self) -> None:
         VerticalScrollInterface.on_interface_hidden(self)
 
+    def _update_restore_charge_opt_status(self, skip_plan: bool) -> None:
+        self.restore_charge_opt.setDisabled(skip_plan)
+        if skip_plan:
+            self.restore_charge_opt.setContent('已启用跳过计划，运行时视为不使用')
```

Ruff (0.15.10)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)